docs/eventing/sugar/README.md

```diff
@@ -0,0 +1,97 @@
+Knative Eventing Sugar Controller will react to special labels and annotations
+to produce or control eventing resources in a cluster or namespace. This allows
+cluster operators and developers to focus on creating fewer resources, and the
+underlying eventing infrastructure is created on-demand, and cleaned up when no
+longer needed.
+
+## Installing
+
+The following command installs the Eventing Sugar Controller:
+
+```bash
+kubectl apply --filename {{< artifact repo="eventing" file="eventing-sugar-controller.yaml" >}}
+```
+
+## Automatic Broker Creation
+
+One way to create a Broker is to manually apply a resource to a cluster using
+the default settings:
+
+```shell
+kubectl create -f - <<EOF
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: default
+  namespace: default
+EOF
+```
+
+There might be cases where automated Broker creation is desirable, such as on
+namespace creation, or on Trigger creation. The Sugar controller enables those
+use-cases:
+
+- When a Namespace is labeled with `eventing.knative.dev/injection=enabled`, the
+  controller will create a default broker in that namespace.
```

Comment:
Unless it's previously defined what name a "default broker" has, I think this should be made more explicit: 

```
the controller will create a default broker named `default` in that namespace.
```

Then subsequent references can either refer to the name of the broker object (`default`) or the name of the concept ("default broker").

Reply:
ok good point.

Reply:
Updated.